x/blob/types/errors.go

```diff
@@ -23,7 +23,7 @@ var (
 	ErrUnsupportedShareVersion        = sdkerrors.Register(ModuleName, 11123, "unsupported share version")
 	ErrZeroBlobSize                   = sdkerrors.Register(ModuleName, 11124, "cannot use zero blob size")
 	ErrMismatchedNumberOfPFBorBlob    = sdkerrors.Register(ModuleName, 11125, "mismatched number of blobs per MsgPayForBlob")
-	ErrInvalidNumberOfPFBInBlobTx     = sdkerrors.Register(ModuleName, 11126, "BlobTx must contain only one MsgPayForBlob")
+	ErrNoPFB                          = sdkerrors.Register(ModuleName, 11126, "no PFB found in transaction")
```

Comment:
Is this error going to stay around or is it temporary?

Reply:
stay around until we support multiple . I felt like `ErrNoPFB` was more descriptive now that we PFB containing transactions can only have one sdk.Msg. I'll also update the description to "no PFB found in blob transaction"

Reply:
What's the timeline for supporting multi blobs

Reply:
Is it that difficult to do?

Reply:
that's already done in #1154, this PR is just a single piece of that PR

Reply:
so whenever we merge all the pieces, likely 3-4 more

Reply:
[question] will this error ever be visible to users that aren't intimately familiar with Celestia? If yes, proposal to avoid using the acronym PFB in favor of the expanded:

```suggestion
	ErrNoPFB                          = sdkerrors.Register(ModuleName, 11126, "no MsgPayForBlob found in transaction")
```

Reply:
@rootulp I edited the above comment to add blob, then added it